pkg/engine/apiPath.go

```diff
@@ -20,6 +21,12 @@ func NewAPIPath(path string) (*APIPath, error) {
 	trimmedPath := strings.Trim(path, "/ ")
 	paths := strings.Split(trimmedPath, "/")
 
+	if paths[0] == "apis" && len(paths) > 7 {
```

Comment:
This `NewAPIPath` looks quite complicated, I wonder if we could replace all with `AbsPath` :thinking:

Reply:
Using `AbsPath` should be the easiest and most straightforward. But I'm thinking there should be a reason for not using `AbsPath`, but I don't know what the reason is.

Reply:
The code here seems to deconstruct the path and reconstruct it in the `String` method...
cc @realshuting @prateekpandey14 @samj1912